wordpress-mobile/WordPress-Android
# Addresses lint issues in AGP 7.2.1 update

This PR addresses the lint issues due to AGP `7.2.1` update in #16834.

* 84598d07a158f9924dab5ffb016605d6360ad226 removes the unused `TextDrawable` widget, so we don't even need to fix the lint issue 🙈 
* d1eeb1ee2c2523ab1fddf80251e5af8dfaf6e67c removes the custom divider and uses the built in `DividerItemDecoration` instead. Funnily enough I worked on the people list 6+ years ago and `DividerItemDecoration` was not available back then (was added a few months after) so we had to build it ourselves. The difference between the regular divider and the custom divider is that the regular divider covers the whole width whereas the custom divider had some space before and after the line. See the screenshots below for comparison. Since we are using the regular divider in a lot of newer screens, this change should bring the design of people list closer to the rest of the app so it's a win-win situation.
* 9988ff81ba20abdf20dab20a6701d78fd0336354 addresses `Obsolete SDK_INT version checks` lint error.

**Before:**

_(Full name and display name are removed and the screenshot was cut in half to make it easier to see)_

<img src="https://user-images.githubusercontent.com/662023/177182718-4d810f71-4fd5-4359-8146-e8dfcd5a4b62.png" width="400">

**After:**

_(Full name and display name are removed and the screenshot was cut in half to make it easier to see)_

<img src="https://user-images.githubusercontent.com/662023/177182730-8e25dff5-cb90-4e2a-b461-c9e707a12b2d.png" width="400">


**To test:**
* Smoke test the app
* Checkout the `People` page from "My Site" -> "People" and verify that the divider looks OK

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

**PR submission checklist:**
